plugins/aea-ledger-ethereum/aea_ledger_ethereum/ethereum.py

```diff
@@ -745,6 +748,14 @@ def load_contract_interface(cls, file_path: Path) -> Dict[str, str]:
                 raise ValueError(f"Contract {file_path} missing key {key}.")
         return contract_interface
 
+    @staticmethod
+    def update_gas_pricing(gas_params: Dict[str, int],) -> Optional[Dict[str, int]]:
+        """Try to update the gas price."""
+        for gas_item in gas_params.keys():
+            gas_params[gas_item] = math.ceil(gas_params[gas_item] * TIP_INCREASE)
```

Comment:
This is probably wrong. The `maxPriorityFeePerGas` should be increased by `TIP_INCREASE` and then the `maxFeePerGas` should be increased by the _amount_ that the `maxPriorityFeePerGas` increased and not the _percentage_.